pep-0514.txt

```diff
@@ -13,8 +13,8 @@ Abstract
 ========
 
 This PEP defines a schema for the Python registry key to allow third-party
-installers to register their installation, and to allow applications to detect
-and correctly display all Python environments on a user's machine. No
+installers to register their installation, and to allow tools and applications
+to detect and correctly display all Python environments on a user's machine. No
 implementation changes to Python are proposed with this PEP.
 
```

Comment:
Maybe 90% is too many, but there's a *big* iceberg of Windows users who have their own internal support teams, aren't using pip/virtualenv, or aren't allowed to comment publicly on _anything_ even to report bugs. And I believe all the IDEs just act as thin wrappers around those tools and will pass bug reports on.

None of the information in that dialog is required, but if parts are missing you'll get a substandard experience. For example, we base some editor features on the language version, so if you pick the wrong one, you'll get the wrong squiggles/etc. If we don't have an executable, we can't run it. They're all fairly obvious IMO.